templates/edit_polycule.jinja2

```diff
@@ -116,27 +118,41 @@
     </div>
 </div>
 <script type="text/javascript">
-    var graph = {{ graph }};
+    var graph = {{ graph or inherited }};
 </script>
 <script src="/static/polycule.js" charset="utf-8"></script>
 <script src="/static/build.js" charset="utf-8"></script>
 
-<form action="/edit/save" method="post" class="wrapper flex" onsubmit="writeGraph();return true;">
-    <textarea name="graph" id="graph-field" class="hidden">{{ graph }}</textarea>
+<form action="{% if creating %}/save {% else %} /edit/save {% endif %}" method="post" class="wrapper flex" onsubmit="writeGraph();return true;">
```

Comment:
Personal preference would be to remove the spaces, but so long as it works, take it or leave it~

Reply:
Oh for sure! Let me know if I need to change it or if GH’s review changes thing handles it

Reply:
Coolcool. Thank you!

Reply:
Oh, and to be clear, I meant the spaces between e.g: `%}/save {%`. I worry that some browsers may interpret the space as a part of the link and go to `/save%20` or something equally silly. I'll get this in a small branch, though!

Reply:
Oh yeah, that’s a very good catch actually! Feel free to fix if you have time, otherwise I’ll PR sometime soon!